fea-rs/src/compile/error.rs

```diff
@@ -81,14 +81,18 @@ struct DiagnosticDisplayer<'a>(&'a DiagnosticSet);
 
 impl DiagnosticSet {
     pub(crate) fn write(&self, f: &mut impl std::fmt::Write, colorize: bool) -> std::fmt::Result {
+        const MAX_MESSAGES_TO_PRINT: usize = 100;
```

Comment:
Optional, CLI and API options to control this would be nice, with the error printing out "pass --blah to see them all" ideally

Reply:
good idea. I've added an option to the `Compiler` struct. Calling fea-rs from the command line is a pretty niche option (fontc doesn't invoke fea-rs via the command line, for instance) and so unless I go with an environment variable this will be something to tune from fontc.